.github/workflows/pypi-release.yml

```
@@ -53,8 +53,27 @@ jobs:
           python -c "import flexaidds as fd; print('flexaidds', fd.__version__, 'HAS_CORE=', getattr(fd, 'HAS_CORE_BINDINGS', False))"
           python -m flexaidds --help
           python -m flexaidds --check-update || true
-          test "$(python -c 'import flexaidds; print(flexaidds.__version__)')" = "2.0.0" \
-            || python -c "import flexaidds as fd; assert fd.__version__, fd.__version__"
+          # Version must be non-empty and match the static PEP 621 field in pyproject.toml
+          # (do not hardcode — that drifted when we shipped 2.0.1/2.0.2).
+          python -c "
+import re
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)